en/companion_computer/holybro_pixhawk_jetson_baseboard.md

```diff
@@ -0,0 +1,1177 @@
+
+Follow the instruction in the ROS2 User guide to [Install ROS 2](../ros2/user_guide.md#install-ros-2) on the Jetson.
+You don't have to install PX4 on the Jetson because we're not using the simulator, but you may wish to install the full desktop so you can use additional ROS 2 packages for further development:
+
+```sh
+sudo apt install -y ros-humble-desktop-full -y
+```
+
+#### Start the XRCE-DDS Agent
+
+Then fetch and build the uXRCE-DDS _agent_ as covered in [Setup the Agent](../ros2/user_guide.md#setup-the-agent) in the ROS 2 User guide.
+
+You will start the agent using different commands, depending on whether the client is configured for a serial or Ethernet connection.
+Assuming the client is set up as defined above:
+
+- Start the agent on `/dev/ttyTHS1` (serial connection):
+
+  ```sh
+  sudo MicroXRCEAgent serial --dev /dev/ttyTHS1 -b 921600
+  ```
+
+- Start the agent on UDP port `8888` (Ethernet):
+
+  ```sh
+  MicroXRCEAgent udp4 -p 8888
+  ```
+
+If your agent and client are running you should see output similar to that below:
+
+![uXRCE-DDS Agent Health check](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/xrce_dds_healthy_connection.png)
+
+#### Starting the Agent on Boot
+
+To run the XRCE-DDS agent each time the Jetson reboots you can make a daemon service to run the agent.
+
+Make a new service file:
+
+```sh
+sudo nano /etc/systemd/system/microxrceagent.service
+```
+
+Paste the following inside:
+
+```plain
+[Unit]
+Description=Micro XRCE Agent Service After=network.target
+[Service]
+14
+Holybro carrier board
+ExecStart=/usr/local/bin/MicroXRCEAgent udp4 -p 8888 Restart=always
+User=root
+Group=root
+ExecStartPre=/bin/sleep 10
+[Install]
+WantedBy=multi-user.target
+```
+
+Save the file and run the following inside terminal
+
+```sh
+sudo systemctl daemon-reload
+sudo systemctl enable microxrceagent.service
+```
+
+Then you can reboot your Jetson board and check if the agent is running in the background:
+
+```sh
+sudo systemctl status microxrceagent.service
+```
+
+If the service is running, you should see output like this:
+
+![XRCE DDS Agent Daemon service](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/xrce_dds_agent_service.png)
+
+You can just do your ROS2 nodes running and the development.
+
+### ROS 2 Sensor Combined Tests
+
+You can test the Client and agent by using the `sensor_combined` example in [Build ROS 2 Workspace](../ros2/user_guide.html#build-ros-2-workspace) (ROS2 User Guide).
+
+::: tip
+[VSCode over SSH](https://code.visualstudio.com/learn/develop-cloud/ssh-lab-machines) enables faster development and application of changes to your ROS 2 code!
+:::
+
+After getting to the point of running the example note:
+
+```sh
+ros2 launch px4_ros_com sensor_combined_listener.launch.py
+```
+
+You should see high frequency sensor messages as the output:
+
+![Sensor node single output](../../assets/companion_computer/holybro_pixhawk_jetson_baseboard/ros_sensor_output.png)
```

Comment:
Can you put the expected output as text rather than a screenshot? Makes easier editing in future if anything changes.